### DISCUSSION TITLE
Spectral rendering RGB space

### DISCUSSION THREAD

**Discussion by vincentlorant** [mitsuba-renderer/mitsuba3#1128] [Q&A]
Hello,

In my ongoing project, I'm employing Mitsuba3 to render a scene using the scalar_spectral variant. I'd like to convert the rendered image into the CIE XYZ color space for further image manipulation. 

My understanding is that Mitsuba renders in Linear RGB space but looking at this [website](http://www.brucelindbloom.com/index.html?Eqn_RGB_XYZ_Matrix.html), I am uncertain about its specific RGB working space. Knowing it would allow me to select the right matrix for the conversion.  I didn't manage to find such information in the documentation so any assistance would be greatly appreciated!

Thanks

**Comment by njroussel** [comment]
Hi @vincentlorant 

The [`hdrfilm` plugin](https://mitsuba.readthedocs.io/en/latest/src/generated/plugins_films.html#high-dynamic-range-film-hdrfilm) has a `pixel_format` that allows you to specify the output color space. One of the options is XYZ.
